examples/counter/lib/main.dart

```diff
@@ -20,7 +23,13 @@ class MyApp extends StatelessWidget {
 }
 
 /// Providers are declared globally and specify how to create a state
```

Comment:
Does it really matter, since the author recently said this 😇
![Capture d’écran 2023-03-02 à 15 27 08](https://user-images.githubusercontent.com/70627850/222456411-10fdaa2e-3cbc-4def-a54f-caf985f20a64.png)

More seriously, I have updated the comments in the file.

Reply:
Hum not fond of them still. I'll update them to my liking, give me a minute.